src/components/CanvasAnimator.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Frame, GraphIndex, LabeledGraph } from '../logic/commonInterfaces';
+import Canvas from './Canvas';
+
+function CanvasAnimator({ graph, frames }
+  : {
+    graph: LabeledGraph,
+    frames: Array<Frame>,
+  }) {
+  return <Canvas
```

Comment:
Why not 

```
return (
    <Canvas nodes={graph[GraphIndex.NODES]} edges={graph[GraphIndex.EDGES]} />
);
```
All in one line, a feast for eyes. Here it looks really weird, at least that's my feeling